data/battle_animation_scripts.py

```diff
@@ -0,0 +1,443 @@
+# List of addresses within the Battle Animation Scripts for the following commands:
+#  B0 - Set background palette color addition (absolute)
+#  B5 - Add color to background palette (relative)
+#  AF - Set background palette color subtraction (absolute)
+#  B6 - Subtract color from background palette (relative)
+# By changing address + 1 to E0 (for absolute) or F0 (for relative), it causes no change to the background color (that is, no flash)
+# If it's commented out, then that background palette change will stay vanilla.
+BATTLE_ANIMATION_FLASHES = [

```

Comment:
This looks ok but adding a big list of addresses is not ideal. I would like to look more into the animation scripts and how much effort it would take to create a nice wrapper for them similar to the enemy scripts.
It would be nice to map the animation names to their script ids. There are more flashes than I anticipated and I think it would be nice to maybe split this up into more options. For example, a setting to only change the worst offenders such as bum rush (should this include disabling poison?). Maybe a photo-sensitivity option that ranges from nothing changed to remove all flashes.

Reply:
Thanks, I see what you have in mind after digging into data/enemy_scripts.py. I pushed a cleaner big-list-of-addresses for now, in which I focused on the more egregious screen flashes (particularly those that flash FF).

Reply:
Per makers-central discussion, I readded flashes and added a "Remove Most" option (renaming Fewer Flashes to Remove Worst)